Final_Project_Paper-AR.qmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Final_Project_Paper"
+Author: "Jeffrey Gilbert"
+format: html
+---
+
+The expansion of imperial powers in the Western Hemisphere has been a foreign policy concern for the United States, which was exemplified by the Monroe Doctrine as well as the Corollary to the Monroe Doctrine.^[fn1: Jeffrey A. Engel, et al., America in the World: A History in Documents from the War with Spain to the War on Terror, (Princeton, NJ: Princeton University Press, 2014), 23-24, 71-72] The fear of foreign influence in Latin America was especially prevalent during the interwar period and World War II. Franklin Delano Roosevelt was keenly aware of Latin America’s distaste for U.S. military intervention, so he decided to deviate from past precedence by taking the approach of being a good neighbor.^[fn2: Irwin F. Gellman, Good Neighbor Diplomacy: United States Policies in Latin America, 1933-1945, (Baltimore: Johns Hopkins University Press, 2019),11-12] Drawing from the digital methods used on the Foreign Policy Bulletin Dataset, which is comprised of publications from the Foreign Policy Bulletin, the interactions driving U.S. foreign policy during Roosevelt’s presidency become more apparent. Even though Latin America’s coverage was underrepresented compared to European or Axis nations, the Foreign Policy Bulletin’s coverage of Latin America suggests that collective security efforts and hemispheric solidarity were central to U.S. foreign policy efforts. Inter-American cooperation and hemispheric solidarity were seen as essential foreign policy measures for protecting the United States from the Axis Powers. 
+
+Over the last century, the Foreign Policy Association (FPA) has been producing content on U.S. foreign policy. Originally established as the League of Free Nations Association, the association’s initial aspirations were to support peace initiatives promulgated by Woodrow Wilson. However, the League of Free Nations Association was short-lived, and the association would shortly be changed to the FPA in 1923, where all matters of U.S. foreign policy fell under their purview of study. Moreover, the FPA had high-ranking U.S. political figures among its incorporators, including Eleanor Roosevelt and John Foster Dulles. In addition to these important incorporators, FPA members also had addresses delivered before them by Franklin D. Roosevelt.^[fn3: Foreign Policy Association, “About FPA,” accessed April 28, 2024, https://www.fpa.org/about/] In 1931, the FPA began producing the Foreign Policy Bulletin, which published articles on international events affecting U.S. foreign policy. The Foreign Policy Bulletin Dataset is comprised of the volumes produced by the Foreign Policy Bulletin between 1936 and 1946.^[fn4: John Mark Ockerbloom, “The Online Books Page: Foreign Policy Bulletin,” accessed April 28, 2024, https://onlinebooks.library.upenn.edu/webbin/serial?id=fpabulletin]
+
+```{r}
+#Load in Libraries
```

Comment:
If you add ```#| echo: false``` to the top of this block it will prevent it from showing up in the final rendered version